managed/GTAVisionExport/VisionExport.cs

```diff
@@ -256,12 +256,13 @@ public void OnTick(object o, EventArgs e)
             if (!runTask.IsCompleted) return;
             if (!postgresTask.IsCompleted) return;
 
-            UI.Notify("going to upload tiff");
+            UI.Notify("going to save tiff");
 
 //            List<byte[]> colors = new List<byte[]>();
             Game.Pause(true);
             Script.Wait(200);
-            GTAData dat = GTAData.DumpData(Game.GameTime + ".tiff", new List<Weather>());
+            var dateTimeFormat = @"dd-MM-yyyy--HH-mm-ss";
+            GTAData dat = GTAData.DumpData(DateTime.UtcNow.ToString(dateTimeFormat) + ".tiff", new List<Weather>());
```

Comment:
UTCNow is not high enough frequency, can have different frames with same timestamp this way.

Reply:
Yes, I noticed that, too. I will add milliseconds here.

Reply:
Just ot of curiosity, in our machine, we have Nvidia GTA 660, and the extraction is in speed 0.7 FPS, on average. 
Which GPU did you use and how many FPS did you obtain for extracted images? @barcharcraz